php/src/vaas/Vaas.php

```diff
@@ -63,18 +68,13 @@ public function Connect(
         string $token,
         ?VaasConnection $vaasConnection = null
     ) {
-        if (isset($vaasConnection)) {
-            $this->_vaasConnection = $vaasConnection;
-        } else {
-            $this->_vaasConnection = new VaasConnection($this->_vaasUrl);
-        }
+        $this->_vaasConnection = isset($vaasConnection) ? $vaasConnection :  new VaasConnection($this->_vaasUrl);
```

Comment:
We can use the null coalescing operator here:

```suggestion
        $this->_vaasConnection = $vaasConnection ?? new VaasConnection($this->_vaasUrl);
```